service/tutorial/order/NewTutorialServices.xml

```diff
@@ -0,0 +1,201 @@
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList"/>
+            <set field="products" from="[]"/>
+
+            <if condition="placedDate">
+                <then>
+                    <set field="orderId" from="order.orderId"/>
+                    <set field="orderName" from="order.orderName"/>
+                    <set field="placedDate" from="order.placedDate"/>
+                    <set field="statusId" from="order.statusId"/>
+                    <set field="orderRevision" from="order.orderRevision"/>
+                    <set field="currencyUomId" from="order.currencyUomId"/>
+                    <set field="productStoreId" from="order.productStoreId"/>
+                    <set field="salesChannelEnumId" from="order.salesChannelEnumId"/>
+                </then>
+            </if>
+         </actions>
+    </service>
+
+    <!-- 5.Finding all Order Header records with the grandTotal greater than 50.-->
+    <service verb="get" noun="OrderHeaders">
+        <out-parameters>
+            <parameter name="orderCount" type="list"/>
+        </out-parameters>
+        <actions>
+            <set field="orderCount" from="[]"/>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+            </entity-find>
+            <iterate list="ordersList">
+                <if condition="grandTotal &gt; 50">
+                    <then>
+                        <set field="orderCount" from="orderCount"/>
+                    </then>
+                </if>
+            </iterate>
+        </actions>
+    </service>
+
+
+    <!--   6.Listing all Order Item details for the order with orderId, 100102 and orderPartSeqId, 01.-->
+    <service verb="get" noun="OrderItems">
+        <in-parameters>
+            <parameter name="orderId"/>
+            <parameter name="orderPartSeqId"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderItemList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderItem" list="orderItemList">
+                <econdition field-name="orderId"/>
+                <econdition field-name="orderPartSeqId"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!--  8.Find all Order Part records assigned to the facility with ZIRET_WH value-->
+
+    <service verb="get" noun="OrderPartData">
+        <in-parameters>
+            <parameter name="facilityId" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderId"/>
+            <parameter name="orderPartSeqId"/>
+            <parameter name="facilityId"/>
+            <parameter name="partName"/>
+            <parameter name="customerPartyId"/>
+        </out-parameters>
+        <actions>
+            <entity-find-one entity-name="mantle.order.OrderPart" value-field="order"/>
+            <set field="orderId" from="order.orderId"/>
```

Comment:
Please explore the way to return only selected fields in entity find, if we can avoid setting the variables separately